### DISCUSSION TITLE
How can I merge without a code review ?

### DISCUSSION THREAD

**Discussion by MahaSegni** [community/community#30451] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, can somebody please help me in how can I merge without review ?
Thank you

**Comment by radar-blip** [comment]
Open a pull request, and close it without reviewing.

**Reply by coltcool999** [reply]
help me pls

**Comment by MahaSegni** [comment]
@linuxlawson I appreciate that, but it's not working. 
Actually, this is about YOLO achievement

**Reply by radar-blip** [reply]
It should work, I dont know why its not working for you.

**Reply by MahaSegni** [reply]
I will try again, thank you

**Reply by radar-blip** [reply]
Keep trying. You will get it. 😄 
I got mine on accident, I dont even want it.

**Comment by MahaSegni** [comment, marked as answer]
✨ It worked, I had to get someone else to review the pull request. ✨

**Comment by msheliga1** [comment]
Any actual DETAILS would be most helpful.  What buttons - located where - need to be pushed to close the pull request?  What buttons - located where - need to be pushed to "merge".  I'm guessing the merge can/must be done from a branch?  (I do agree that one cannot "approve" their own pull request. But the original question (and my issue) involves merging code without doing a review.
